docs/sidebars.js

```diff
@@ -270,17 +270,30 @@ module.exports = {
                         slug: 'references/admin',
                     },
                     items: [
-                        {
-                            type: 'doc',
-                            id: 'references/personal_tokens',
-                            label: 'Project Settings',
-                        },
                         'references/usage-analytics',
                         'references/roles',
                         'references/groups',
                         'references/user-attributes',
                     ],
                 },
+                {
+                    type: 'category',
```

Comment:
Fix document sidebar location

Reply:
@jpetey75 Let me know if you agree with this.

Reply:
@ZeRego maybe it can just go under `References > Lightdash Development`? I don't think there are very many items to put into a `User` category 

The other option is just to put a `User Settings` under the `Admin` category, or even at the top level without a category, then we just put all user settings there. 

I might move to `Lightdash Development` next time I make a docs change, but it's not super important IMO